### DISCUSSION TITLE
Google Maps Key

### DISCUSSION THREAD

**Discussion by rafael844** [sepinf-inc/IPED#247] [Q&A] [answered]
Is there a way to set the google maps javascript key in config file?

**Comment by lfcnassif** [comment, marked as answer]
Currently not, because your config files are sent together with your reports, and that could leak your key. But the key should be saved locally in your computer after the first time it is used. Isn't it being populated automatically in the key dialog in a second run of the GUI?

**Comment by rafael844** [comment]
Yes, its being populated, but who do the analisys is other people from the same organization, in other computers. But its true, it would be a leak.

Thank you.
